packages/react/avatar/src/Avatar.tsx

```diff
@@ -116,35 +119,47 @@ AvatarFallback.displayName = FALLBACK_NAME;
 
 /* -----------------------------------------------------------------------------------------------*/
 
+function setImageSrcAndGetInitialState(image: HTMLImageElement, src?: string): ImageLoadingStatus {
```

Comment:
Can we rename this to `resolveLoadingStatus`?

Reply:
@chaance done! I also rebased with main since there's some conflict with the referrerPolicy change